src/server_manager/web_app/digitalocean_app/verify_account_app.ts

```diff
@@ -0,0 +1,176 @@
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import '@polymer/polymer/polymer-legacy.js';
+import '@polymer/iron-pages/iron-pages.js';
+import '../ui_components/outline-step-view.js';
+
+import {css, customElement, html, LitElement, property} from 'lit-element';
+
+import {makePublicEvent} from '../../infrastructure/events';
+import {sleep} from '../../infrastructure/sleep';
+import {DigitalOceanAccount} from '../../model/digitalocean_account';
+import {COMMON_STYLES} from '../ui_components/cloud-install-styles';
+import {OutlineNotificationManager} from '../ui_components/outline-notification-manager';
+
+@customElement('digital-ocean-verify-account-app')
+export class DigitalOceanVerifyAccountApp extends LitElement {
+  @property({type: Function}) localize: Function;
+  @property({type: String}) currentPage: string;
+  @property({type: Object}) notificationManager: OutlineNotificationManager = null;
+
+  private cancelled = false;
+  private activatingAccount = false;
+
+  static get styles() {
+    return [
+      COMMON_STYLES, css`
+      :host {
+      }
+      .container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: 100%;
+        align-items: center;
+        padding: 132px 0;
+        font-size: 14px;
+      }
+      .card {
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+        justify-content: space-between;
+        margin: 24px 0;
+        padding: 24px;
+        background: var(--background-contrast-color);
+        box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.14), 0 2px 2px 0 rgba(0, 0, 0, 0.12), 0 1px 3px 0 rgba(0, 0, 0, 0.2);
+        border-radius: 2px;
+      }
+      @media (min-width: 1025px) {
+        paper-card {
+          /* Set min with for the paper-card to grow responsively. */
+          min-width: 600px;
+        }
+      }
+      .card p {
+        color: var(--light-gray);
+        width: 100%;
+        text-align: center;
+      }
+      .card paper-button {
+        color: var(--light-gray);
+        width: 100%;
+        border: 1px solid rgba(255, 255, 255, 0.12);
+        border-radius: 2px;
+      }
+      .card paper-button[disabled] {
+        color: var(--medium-gray);
+        background: transparent;
+      }
+      /* Mirror images */
+      :host(:dir(rtl)) .mirror {
+        transform: scaleX(-1);
+      }
+    `
+    ];
+    // TODO: RTL
+  }
+
+  render() {
+    return html`
+    <iron-pages id="pages" attr-for-selected="id" selected="${this.currentPage}">
```

Comment:
Get rid of iron-pages. No need when using lit-element. Just return the block you need.

Reply:
perhaps you make currentPage be of html`` template type.
Then you can have this.currentPage = verifyEmailHtml() in the start() method.